{{cookiecutter.project_slug}}/.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,95 @@
+name: Github-CI
```

Comment:
As this file lives in the `.github` folder, I think we could call it `ci.yml` and name it `CI` here.

Reply:
Ok. I'll do that.